api/src/main/java/org/pmiops/workbench/actionaudit/bucket/BucketAuditEntry.java

```diff
@@ -0,0 +1,51 @@
+package org.pmiops.workbench.actionaudit.bucket;
```

Comment:
This class represents one entry returned from BQ. Each entry is grouped by the pet account, and has the sum of all file lengths created within the last 24 hours